COMMUNITY_BADGES.md

```diff
@@ -0,0 +1,67 @@
+# OPA Community Badges
+
+Hello OPA Community Members, we thank you for taking the time to be a part of the community and for all of the work you’ve put in. As thanks, the OPA community now offers badges to recognize your contributions to the community.
+
+As of right now, these badges are simple tokens of appreciation that we will proudly display in the OPA GitHub repository. As the program develops, we will provide digital badges and certificates for members to display on their Linkedin and Twitter accounts.
+
+## Contributor Badge
+
+We appreciate contributions to the OPA project, no matter how big or small. So if you’ve contributed a dozen lines of code or corrected a typo in the docs, let us know!
+
+### Requirements
+
+- 1 Merged PR
+
+## Spokesperson Badge 
+
+For community members that like to create OPA content, share your work to receive the OPA Spokesperson Badge.
+
+### Requirements
+
+One or more pieces of publically posted content such as any of the following (Available for non-vendor related content only):
+
+- Conference Talk
+- Meetup presentation
+- Online Video Recording
+- Technical blog or OPA tutorial 
+
+## Super User Badge 
+
+This badge is for members that know the value of running OPA in production. Add your organization to the ADOPTERS.md file, then share a quick snippet about how OPA is used in your production environment to earn this badge. 
+  
+### Requirements
+
+- Deploy or Support OPA in a production environment
+- Add your organization to the ADOPTERS.md file
+
+### Benefits
+
+- Access to a private OPA Super Users Slack Channel
+- Support writing an OPA CFP to grow your portfolio
+
+## Community Champion Badge
+
+Our OPA Community Champions badges are for active members that want to create a measurable impact in the community. Apply for this badge if you believe in the OPA mission to unify policy authorization and wish to help the OPA community grow and expand.
+
+### Requirements
+
+Actively help and support the OPA community. Some places we will look for contributions are:
+
+- Slack
+- GitHub Discussions 
+- Stack Overflow
+- Meetups
+
+Being a champion in the OPA community can take many shapes and forms; as such, we do not want to limit this award to a static number of posts or comments. Instead, we are looking for members who take an active role in helping to build the community, or we look for members that want to have a more active role. 
+
+### Benefits
+
+- Access to a private OPA Champions Slack Channel
+- Help from the OPA team to assist with community issues
+- Increased support for your OPA projects
+
+## Application Process
+
+To apply for any of the OPA badges, fill out the form at: https://airtable.com/shrrd45fRi7yO2k6i
+
+For any questions, please reach out to peteroneilljr@styra.com or @peteroneilljr in [Slack](https://slack.openpolicyagent.org/).
```

Comment:
Given how I think we can assume/require Slack membership, I think we can skip the email address here?

Reply:
Yea most likely not necessary, but I think it's fine to leave it.